db/file_indexer.h

```diff
@@ -58,7 +58,8 @@ class FileIndexer {
                    std::vector<FileMetaData*>* const files);
 
   enum {
-    kLevelMaxIndex = std::numeric_limits<int32_t>::max()
+    // MSVC version 1800 still does not have constexpr for ::max()
+    kLevelMaxIndex = INT32_MAX
```

Comment:
Can you create a const in port.h and still use std::numeric_limits<int32_t>::max() for other platforms? I'm concerned some platform might not have INT32_MAX.